doc/api/vm.md

```diff
@@ -164,22 +164,22 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
   * `context` {Object} The [contextified][] object as returned by the
     `vm.createContext()` method, to compile and evaluate this `Module` in.
   * `lineOffset` {integer} Specifies the line number offset that is displayed
-    in stack traces produced by this `Module`.
+    in stack traces produced by this `Module`. **Default:** `0`.
   * `columnOffset` {integer} Specifies the column number offset that is
-    displayed in stack traces produced by this `Module`.
+    displayed in stack traces produced by this `Module`. **Default:** `0`.
   * `initializeImportMeta` {Function} Called during evaluation of this `Module`
-    to initialize the `import.meta`. This function has the signature `(meta,
-    module)`, where `meta` is the `import.meta` object in the `Module`, and
-    `module` is this `vm.SourceTextModule` object.
-  * `importModuleDynamically` {Function} Called during evaluation of this
-    module when `import()` is called. This function has the signature
-    `(specifier, module)` where `specifier` is the specifier passed to
-    `import()` and `module` is this `vm.SourceTextModule`. If this option is
-    not specified, calls to `import()` will reject with
-    [`ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`][]. This method can return a
-    [Module Namespace Object][], but returning a `vm.SourceTextModule` is
-    recommended in order to take advantage of error tracking, and to avoid
-    issues with namespaces that contain `then` function exports.
+    to initialize the `import.meta`.
+    * `meta` {import.meta}
+    * `module` {vm.SourceTextModule}
+  * `importModuleDynamically` {Function} Called during evaluation of this module
+    when `import()` is called. If this option is not specified, calls to
+    `import()` will reject with [`ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`][].
+     * `specifier` {string} specifier passed to `import()`
+     * `module` {vm.SourceTextModule}
+     * Returns: {Module Namespace Object|vm.SourceTextModule} Returning a
+       `vm.SourceTextModule` is recommended in order to take advantage of error
+       tracking, and to avoid issues with namespaces that contain `then`
+       function exports.
```

Comment:
I would suggest including a note here saying something like:

"This option is part of the experimental API for the `--experimental-modules` flag, and should not be considered stable."

Reply:
Sorry I copied the wrong one, this should apply to the whole `vm.sourceTextModule` section.

The note above should apply only for the `vm.Script` option.

Reply:
Added in `vm.Script()` constructor and all concerned `vm.runIn...()` methods.

Reply:
`vm.sourceTextModule` already has axperimental note.